scripts/rollup/bundles.js

```diff
@@ -169,19 +169,24 @@ const bundles = [
       sourceMap: false,
     },
     entry: 'src/renderers/dom/ReactDOMServer.js',
-    externals: ['prop-types', 'prop-types/checkPropTypes'],
+    externals: [
+      'prop-types',
+      'prop-types/checkPropTypes',
+      'create-react-class/factory',
+    ],
     fbEntry: 'src/renderers/dom/ReactDOMServer.js',
     hasteName: 'ReactDOMServerStack',
     isRenderer: true,
     label: 'dom-server',
     manglePropertiesOnProd: false,
     name: 'react-dom/server',
     paths: [
+      'src/isomorphic/**/*.js',
       'src/renderers/dom/**/*.js',
       'src/renderers/shared/**/*.js',
-
       'src/ReactVersion.js',
       'src/shared/**/*.js',
+      'src/addons/**/*.js',
     ],
   },
   // TODO: there is no Fiber version of ReactDOMServer.
```

Comment:
Let's take this chance to duplicate the entry above to a new flat bundle entry for the new `ReactDOMStreamServer` module.